### PR TITLE
DSD-874: Minor update to use a design token name for the Notification font size

### PR DESCRIPTION
**Pull request by EdwinGuzman**
Fixes JIRA ticket [DSD-874](https://jira.nypl.org/browse/DSD-874)

## This PR does the following:

- Replaces `"-1"` with `"text.caption"` for the `Notification`'s font size. Internally, they are the same value so no visual change.

## How has this been tested?

Storybook.

## Accessibility concerns or updates

- N/A

### Checklist:

- [ ] I have updated the Storybook documentation accordingly.
- [ ] I have added relevant accessibility documentation for this pull request.
- [ ] All new and existing tests passed.

### Front End Review:

- [ ] View [the example in Storybook]()

### CODE REVIEW

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr943-gnydi1pq0dy80tiwvu1pevmwkinujq49.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/624f45d77dc9604ba3dfaf09

File: src/theme/components/notification.ts [hunk 1]
```diff
@@ -13,7 +13,7 @@ const Notification = {
     return {
       bg,
       display: "flex",
-      fontSize: "-1", // slightly smaller than the default size
+      fontSize: "text.caption", // slightly smaller than the default size
```
Comment (oliviawongnyc) [inline src/theme/components/notification.ts:16]: Do you still need the comment?

Comment (bigfishdesign13) [reply src/theme/components/notification.ts:16]: Yes, let’s keep the comment to show that the `fontSize` value was intentional.

**Review by bigfishdesign13 — APPROVED** [review]
Indeed.

**Pull request merged**